saleor/discount/utils.py

```diff
@@ -1707,7 +1715,7 @@ def create_or_update_discount_object_from_order_level_voucher(order):
         return
 
     voucher = order.voucher
-    if not is_order_level_voucher(voucher):
+    if not (is_order_level_voucher(voucher) or is_shipping_voucher(voucher)):
```

Comment:
Could we change it to:
```suggestion
    if not is_order_level_voucher(voucher) or not is_shipping_voucher(voucher):
```
? IMO it's clearer, for the first sight I was thinking that you are returning then it's a shipping voucher 😄

Reply:
Done